samples/drivers/watchdog/sample.yaml

```diff
@@ -22,6 +22,8 @@ tests:
       - s32z2xxdc2/s32z270/rtu1
       - s32z2xxdc2@D/s32z270/rtu0
       - s32z2xxdc2@D/s32z270/rtu1
+      - s32k5xxcvb/s32k566/m7
```

Comment:
I'd suggest renaming `sample.drivers.watchdog.s32z270dc2_r52` to `sample.drivers.watchdog.nxp_s32_swt` and adding s32k5 platform to this build-only scenario

Reply:
I've updated